src/protagonist/API.Client/DlcsClient.cs

```diff
@@ -317,7 +317,38 @@ public async Task<CustomerQueue> GetQueue()
         var queue = await response.ReadAsHydraResponseAsync<CustomerQueue>(jsonSerializerSettings);
         return queue;  
     }
+    
+    public async Task<IEnumerable<NamedQuery>> GetNamedQueries(bool includeGlobal)
+    {
+        var url = $"customers/{currentUser.GetCustomerId()}/namedQueries";
+        var response = await httpClient.GetAsync(url);
+        var namedQueries = await response.ReadAsHydraResponseAsync<HydraCollection<NamedQuery>>(jsonSerializerSettings);
+
+        return includeGlobal
+            ? namedQueries.Members
+            : namedQueries.Members.Where(nq => nq.Global == false);
+    }
+
+    public async Task DeleteNamedQuery(string namedQueryId)
+    {
+        var url = $"customers/{currentUser.GetCustomerId()}/namedQueries/{namedQueryId}";
+        await httpClient.DeleteAsync(url);

```

Comment:
Feels like we should report success/failure back to use for this? (same goes for update too).

Could be frustrating to hit 'Save' and think it's went through but nothing lets you know it's failed.